Samples/PublicSamples/RecordingBot/src/RecordingBot.Model/Extension/ParticipantExtension.cs

```diff
@@ -56,104 +30,43 @@ public ParticipantExtension(IParticipant participant)
 
-        /// <summary>
-        /// Gets the created date time of this resource.
-        /// </summary>
-        /// <value>The created date time.</value>
         public DateTimeOffset CreatedDateTime { get; private set; }
 
-        /// <summary>
-        /// Gets the client.
-        /// </summary>
-        /// <value>The client.</value>
         [JsonIgnore]
         public ICommunicationsClient Client { get; private set; }
 
-        /// <summary>
-        /// Gets the graph client.
-        /// </summary>
-        /// <value>The graph client.</value>
         [JsonIgnore]
         public IGraphClient GraphClient { get; private set; }
 
-        /// <summary>
-        /// Gets the graph logger.
-        /// </summary>
-        /// <value>The graph logger.</value>
         [JsonIgnore]
         public IGraphLogger GraphLogger { get; private set; }
 
-        /// <summary>
-        /// Gets the resource path for this collection.
-        /// </summary>
-        /// <value>The resource path.</value>
         public string ResourcePath { get; private set; }
 
-        /// <summary>
-        /// Gets the stateful participant resource.
-        /// </summary>
-        /// <value>The resource.</value>
-        /// <exception cref="NotImplementedException"></exception>
         [JsonIgnore]
         object IResource.Resource => throw new NotImplementedException();
 
-        /// <summary>
-        /// Event fired when this resource has been updated.
-        /// </summary>
         public event ResourceEventHandler<IParticipant, Participant> OnUpdated;
 
-        /// <summary>
-        /// Deletes this participant asynchronously.
-        /// </summary>
-        /// <param name="handleHttpNotFoundInternally">If the <see cref="T:Microsoft.Graph.Communications.Calls.IParticipant" /> is already gone, whether to handle the exception gracefully or not.</param>
-        /// <param name="cancellationToken">The <see cref="T:System.Threading.CancellationToken" /> for the request.</param>
-        /// <returns>The <see cref="T:System.Threading.Tasks.Task" /> that completes after the request has been sent.
-        /// The completion of this task does not guarantee deletion. Confirmation of
-        /// deletion comes as a notification and can be subscribed by IParticipant.OnUpdated and
-        /// IParticipantCollection.OnUpdated</returns>
-        /// <exception cref="NotImplementedException"></exception>
         public Task DeleteAsync(bool handleHttpNotFoundInternally = false, CancellationToken cancellationToken = default)
         {
             throw new NotImplementedException();
         }
 
-        /// <summary>
-        /// Performs application-defined tasks associated with freeing, releasing, or resetting unmanaged resources.
-        /// </summary>
-        /// <exception cref="NotImplementedException"></exception>
         public void Dispose()
         {
             throw new NotImplementedException();
         }
 
-        /// <summary>
```

Comment:
This does not seem like a Ghost Doc

Reply:
True, but still doesn't improve the info to noise ratio of the comments, so I'll stick to removing this